src/tools/run-make-support/src/diff/mod.rs

```diff
@@ -58,15 +66,24 @@ impl Diff {
         self
     }
 
+    pub fn normalize(&mut self, regex: &str, replacement: &str) -> &mut Self {
```

Comment:
Missing a test in [`src/tools/run-make-support/src/diff/tests.rs`](https://github.com/rust-lang/rust/blob/f3530cf003745af53f82c8286ab63597b615095e/src/tools/run-make-support/src/diff/tests.rs)

Reply:
Added it!